src/components/MessageList/MessageList.js

```diff
@@ -32,6 +32,7 @@ class MessageList extends PureComponent {
     this.state = {
```

Comment:
Adding this file only to make it easier to see diff. This file will be reverted after the review